### PR TITLE
Remove addons and Mixer from install templates

### PR DESCRIPTION
**Pull request by howardjohn**
This executes the plan of https://github.com/istio/istio/issues/25333 and https://github.com/istio/istio/issues/23583

Note: the dashboards are copied currently. If https://github.com/istio/release-builder/pull/307 is merged first, we don't need to do this.

### CODE REVIEW

**Comment by douglas-reid** [comment]
@howardjohn what can I do to help get this in?  once this is in, I believe the e2e tests for my massive Mixer PR will be unblocked.

**Comment by howardjohn** [comment]
Waiting on https://github.com/istio/istio/pull/25719 then will rebase and should be good to go. https://github.com/istio/istio/pull/25723 would be nice but its also a 1 liner so can merge it into this one. Other than that we can start tracking down the huge group of owners that are needed to approve this

File: tests/integration/telemetry/util.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	res, err := fhttp.RunHTTPTest(&opts)
+	if err != nil {
+		t.Fatalf("Generating traffic via fortio failed: %v", err)
+	}
+	return res
+}
+
+func AllowRuleSync(t *testing.T) {
+	t.Log("Sleeping to allow rules to take effect...")
+	time.Sleep(15 * time.Second)
```
Comment (howardjohn) [inline tests/integration/telemetry/util.go:81]: This file is copy+pasted from mixer - if we want to change the logic lets do it in another PR

**Comment by howardjohn** [comment]
Adding release-notes-none since the release notes test has a bug and I want a green checkmark... this does have a release note

/retest

**Review by JimmyCYJ — APPROVED** [review]

**Review by douglas-reid — APPROVED** [review]
approved. i've made the same testing changes in my mixer removal pr.

**Review by mandarjog — APPROVED** [review]
/lgtm

**Comment by mandarjog** [comment]
These two PRs are going to create many merge conflicts for any in-flight PRs that want to get into 1.7.

**Review by sdake — APPROVED** [review]
I thought we were keeping Prometheus with a 6-hour timer to feed other addons (or other Prometheus instances). We may need to reintroduce Prometheus, but it seems like this is blocking alot of other's work, so I'll approve now. Hopefully, the docs will be ready for 1.8 :)

Cheers,
-steve
